### PR TITLE
Deprecate old selectDropdown e2e helper

### PR DESCRIPTION
**Pull request by raquelromano-gov**
Was doing some accessibility testing stuff and noticed this helper can be deprecated now.

### CODE REVIEW

File: test/util/nightwatch-commands/selectDropdown.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /**
- * For chaining selecting dropdowns
- * Note: This takes the field name, not the whole selector.
+ * Change select value and trigger change event programatically. This
+ * is necessary because long select boxes tend to render offscreen,
+ * causing Selenium to fail in unexpected ways.
```
Comment (cvalarida) [inline test/util/nightwatch-commands/selectDropdown.js]: Ooh, good point! I was wondering what the deal was.

**Review by cvalarida — APPROVED** [review]
Thanks for doing this!

File: test/util/nightwatch-commands/selectDropdown.js (hunk 1 above)
Comment (raquelromano-gov) [inline test/util/nightwatch-commands/selectDropdown.js]: I just swiped this comment from the old function, but yes, good to know.

Comment (cvalarida) [inline test/util/nightwatch-commands/selectDropdown.js]: Shows how much I pay attention! ¯\\_(ツ)_/¯

**Pull request merged**